src/screens/About/About.tsx

```diff
@@ -141,15 +137,9 @@ const About = () => {
             <MarkdownContent source={aboutContent.impactContent} />
           </SectionContent>
           <ButtonContainer>
-            <Button to="/case-studies" cta="View case studies" />
+            <Button href="https://actnowcoalition.org" cta="Learn more" />
```

Comment:
Nit: Can you link to https://www.actnowcoalition.org/ instead (with www) to avoid a redirect?